### ISSUE TITLE
Highmountain Quest: "Rite of Blood"

### ISSUE THREAD

**Issue by Lhyron** [Valhalla-Gaming/Tracker#8472]
**Describe the issue you're having**:
Marakhan appears when you approach Navarrogg. However, he doesn't kill Oakin Ironbull and can't be attacked and killed. It says "unvalid target".

**I've been able to identify the origin of the bug. A NPC called "Rivermane emissary" is in the area. When Marakhan appears, he directly cast his spell on this Rivermane emissary that doesn't kill him instead of Oakin Ironbull. Therefore, Oakin Ironbull is still alive and Marakhan is an "unvalid target".**

**Explain how you expect it work**:
No Rivermane emissary should be. Marakhan should be killable once Oakin Ironbull is dead. 
**Steps to reproduce the problem**:

1. Take the quest
2. Go next to Navarrogg
3. Wait & See if Marakhan can be attacked

**Links to Wowhead, YouTube, etc**:
http://www.wowhead.com/quest=39860/rite-of-blood
https://www.youtube.com/watch?v=O4GuHUJ-8sE
http://www.noelshack.com/2017-03-1484610052-capture-d-ecran-2017-01-16-a-23-25-21.png

**Comment by Guybrush-VR** [comment]
Thank you for your report, I see what's going on now!
Fun fact about Rivermane Emissary: He is in fact the Oakin Ironbull of another player. Same goes for you, you can see your Oakin Ironbull but other players will see Rivermane Emissary.

**Comment by Lhyron** [comment]
Ok, I see. Then it must be the same with the Stonedark Drogbar which are Navarrogg of other players. The zone is full of those.

**Comment by Guybrush-VR** [comment]
Yes, exactly. I'll fully rework this quest once we have a new feature that is required to make this quest retail-like. Thank you for your patience during this time! :sunglasses:

**Comment by Proklet** [comment]
I have an update to this. As of now, the quest mob Marakhan can't be targeted by pets or their abilities. I am a Survival Hunter, nothing I do can make my pet attack him or use the most basic Growl ability to take the aggro away from me. Which brings me to the next thing I wanted to mention:
This mob should be soloed, but I am not sure if it's a phasing thing, but, nobody other than me can even see the mob, even if we are in party. Like it's not there or in a different phase. It;s a possibility that my pet also "can't see him" in similar fashion.
In any case, this mob can't be killed other than by soloing him, but he's an elite, on level 101 he has 3.5 million HP, and hits me (Survival Hunter) for around 70-80k (I have around 500k HP). There is NO way I can dish out enough damage to kill him quickly enough on my own, especially with my pet being unable to take aggro away from me. I suspect similar problems occur with other specs that rely on pets.
I do have to note that all my abilities work perfectly fine on Marakhan, but his Doom Blade ability (I think that's what it's called) deals only 150 damage - most probably bugged.
